hardware_interface/src/resource_manager.cpp

```diff
@@ -60,7 +60,7 @@ auto trigger_and_print_hardware_state_transition =
   }
   else
   {
-    RCUTILS_LOG_INFO_NAMED(
+    RCUTILS_LOG_WARN_NAMED(
```

Comment:
```suggestion
    RCUTILS_LOG_ERROR_NAMED(
```

Should this be in error or warning?

Reply:
you are right!